### PR TITLE
Proposed Version 1

### PR DESCRIPTION
**Pull request by philschanely**
Please see the instructions and contained files. Feel free to try out the challenge.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (monkeypox8) [inline .gitignore:1]: We may also want to add:
```
.DS_Store
package-lock.json
```

File: README.md [hunk 2]
```diff
@@ -1 +1,70 @@
-# uxd_challenge
\ No newline at end of file
+# Kajabi UXD Candidate Code Challenge
+
+Welcome! We're excited you're interested in joining the Kajabi UX Developer (UXD) team! This code challenge gives you an opportunity to show us where you are in relation to crucial skills for day to day work at Kajabi.
+
+## Your challenge, should you choose to accept it...
+
+The overall goal of this challenge is to create a prototype component using HTML and CSS based on a provided design mockup. Please don't spend more than 4 hours on the challenge.
```
Comment (QuintonJason) [inline README.md:7]: I'm wondering if he mention of time in hours is going to back fire. Maybe something like, don't spend more than 2 or 3 days. Even though they are applying, these days are unexpected and we don't want the candidate to get the wrong impressive of our culture

**Review by monkeypox8 — APPROVED** [review]
👍🏼 LGTM!

File: README.md (hunk 2 above)
Comment (joedakroub) [reply README.md:7]: This is a good point. I can see changing this days or dropping the statement about time completely. I expect there would be a submission date in an email so maybe that is sufficient?

**Review by joedakroub — APPROVED** [review]
This looks great. The example has enough challenge to it and is very relevant. Nice work! 👍🏼

**Pull request merged**